notebooks/00_quick_start/sar_movielens_with_azureml.ipynb

```diff
@@ -0,0 +1,668 @@
+      "Uploaded aml_data/movielens_10m_data.pkl, 4 files out of an estimated total of 5\n",
+      "Uploaded aml_data/movielens_20m_data.pkl, 5 files out of an estimated total of 5\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "$AZUREML_DATAREFERENCE_7d80fa88a67148b298d22b6d448c507c"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "DATA_DIR = 'aml_data'\n",
+    "TARGET_DIR = 'movielens'\n",
+    "\n",
+    "os.makedirs(DATA_DIR, exist_ok=True)\n",
+    "\n",
+    "# download dataset\n",
+    "data = movielens.load_pandas_df(\n",
+    "    size=MOVIELENS_DATA_SIZE,\n",
+    "    header=['UserId','MovieId','Rating','Timestamp']\n",
+    ")\n",
+    "\n",
+    "# upload dataset to workspace datastore\n",
+    "data_file_name = \"movielens_\" + MOVIELENS_DATA_SIZE + \"_data.pkl\"\n",
+    "data.to_pickle(os.path.join(DATA_DIR, data_file_name))\n",
+    "\n",
+    "ds = ws.get_default_datastore()\n",
+    "\n",
+    "ds.upload(src_dir=DATA_DIR, target_path=TARGET_DIR, overwrite=True, show_progress=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create or Attach Azure Machine Learning Compute \n",
+    "\n",
+    "We create a cpu cluster as our **remote compute target**. If a cluster with the same name already exists in your workspace, the script will load it instead. You can read [Set up compute targets for model training](https://docs.microsoft.com/en-us/azure/machine-learning/service/how-to-set-up-training-targets) to learn more about setting up compute target on different locations. You can also create GPU machines when larger machines are necessary to train the model.\n",
+    "\n",
+    "According to Azure [Pricing calculator](https://azure.microsoft.com/en-us/pricing/calculator/), with example VM size `STANDARD_D2_V2`, it costs a few dollars to run this notebook, which is well covered by Azure new subscription credit. For billing and pricing questions, please contact [Azure support](https://azure.microsoft.com/en-us/support/options/).\n",
+    "\n",
+    "**Note**:\n",
+    "- 10m and 20m dataset requires more capacity than `STANDARD_D2_V2`, such as `STANDARD_NC6` or `STANDARD_NC12`. See list of all available VM sizes [here](https://docs.microsoft.com/en-us/azure/templates/Microsoft.Compute/2018-10-01/virtualMachines?toc=%2Fen-us%2Fazure%2Fazure-resource-manager%2Ftoc.json&bc=%2Fen-us%2Fazure%2Fbread%2Ftoc.json#hardwareprofile-object).\n",
+    "- As with other Azure services, there are limits on certain resources (for eg. AmlCompute quota) associated with the Azure Machine Learning service. Please read [these instructions](https://docs.microsoft.com/en-us/azure/azure-supportability/resource-manager-core-quotas-request) on the default limits and how to request more quota.\n",
+    "\n",
+    "#### Learn more about Azure Machine Learning Compute\n",
```

Comment:
The details aren't showing up properly here

Reply:
Details are not showing in github preview but works for me in jupyter notebook